liquidcore/home/oauth2_validator.py

```diff
@@ -11,4 +11,5 @@ def get_additional_claims(self, request):
             "email": fake_user_email,
             "first_name": request.user.first_name,
```

Comment:
as discussed in other PR, let's try to give defaults here with the username, in case the field is empty/too short

Reply:
use of username is passed now, I hadn't idea of it at the time, last thing we should protect ourselves from is if someone has a username < 2 letters, if you can guide me trough logging + returning an error I can add that